tests/middleware/test_retries.py

```diff
@@ -1,33 +0,0 @@
-import remoulade
```

Comment:
why was this removed?

Reply:
It tested a feature that was not useful, using default_before that was deleted.